test/catalog/catalog_test.cpp

```diff
@@ -116,40 +117,6 @@ TEST_F(CatalogTests, CreatingTable) {
                         ->GetColumnObject(1)
                         ->GetColumnName());
   txn_manager.CommitTransaction(txn);
-  // EXPECT_EQ(5, time_stamp);
```

Comment:
Would it be possible for you to check the table count? Currently it should be 8, correct? Similarly to the index as well? 
So this way we can check that people always account for the additional catalog tables/indexes they create.